lib/engine/game/g_1893/buy_eva_from_other_player.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module BuyEvaFromOtherPlayer
+  def buy_eva_from_other_player(entity, company, price)
+    raise GameError, "Only #{@game.eva.name} can be bought from other player" unless company == @game.eva
+
+    buyer = spender(entity)
+    @log << "Buyer #{buyer.name}, Entity #{entity.name}, Company #{company.name}"
+    owner = company.owner
+    company.owner = buyer
+    owner&.companies&.delete(company)
```

Comment:
why would owner be nil, if owner is nil line 15 will crash

Reply:
Probably some cut & paste code from a context where company might have no owner. 
Right, removed &.